cmd/clusterctl/cmd/move.go

```diff
@@ -78,6 +80,10 @@ func init() {
 		"Write Cluster API objects and all dependencies from a management cluster to directory.")
 	moveCmd.Flags().StringVar(&mo.fromDirectory, "from-directory", "",
 		"Read Cluster API objects and all dependencies from a directory into a management cluster.")
+	moveCmd.Flags().Float32Var(&mo.restQPS, "rest-qps", 0,
+		"Maximum QPS from the REST client to Kubernetes.")
+	moveCmd.Flags().IntVar(&mo.restBurst, "rest-burst", 0,
+		"Maximum burst for the REST client to Kubernetes.")
```

Comment:
For move we have two kubeconfig/clients (kubeconfig and to-kubeconfig). Should we have flags to set qps and burst for the second one as well?

Reply:
Yeah that seems reasonable